web/src/views/MainView.coffee

```diff
@@ -86,7 +86,9 @@ export default
 		GitInput would have done the git call ###
 		run_log = (###* @type string ### log_args) =>
 			sep = '^%^%^%^%^'
-			log_args = log_args.replace(" --pretty=VSCode", " --pretty=format:'#{sep}%h#{sep}%an#{sep}%ae#{sep}%at#{sep}%D#{sep}%s'")
+			log_args = log_args.replace(" --pretty=VSCode", " --pretty=format:\"#{sep}%h#{sep}%an#{sep}%ae#{sep}%at#{sep}%D#{sep}%s\"")
+			stash_refs = try await git 'reflog show --format="%h" stash' catch then ""
```

Comment:
This doesn't work on my machine:
![screenshot](https://waritschlager.de/share/C523380770B3ECB6DE84192C96268440.png)
as can be seen in the screenshot, the output of `git reflog show` is a newline-delimited list of commit hashes which is then inserted verbatim into the main git log command, but newlines start new commands. Pretty scary when I noticed. Apparently, subshell `$()` fixes this.

I am surprised to see this was no problem for you on Windows though, it seems that on there, 1 string = 1 command, no matter its contents.

Either way, we need another `.replaceAll('\n', ' ')` somewhere here or something similar to remove the newlines, then I can merge your PR.

Reply:
Ok - let me have a look.

Reply:
Should be fixed now.  I was surprised because I test using WSL as well, but the issue was I tested with only a single stash present in the repo.  Having multiple stashes doesn't work on Windows either, although it's less catastrophic (just silently drops the rest of the command it seems).

Reply:
great